.github/workflows/ci.yml

```diff
@@ -8,7 +8,23 @@ on:
   schedule:
     - cron: '0 0 * * 4'
 
+permissions:
+  contents: read
```

Comment:
This workflow only checks data without altering it; stating this explicitly is more secure.